tests/core/test_stuff.py

```diff
@@ -466,3 +468,21 @@ def is_in_calsteps(x: mass2.core.RecipeStep, steps: mass2.core.Recipe) -> bool:
 
     with pytest.raises(ValueError):
         steps.trim_dead_ends("this field doesn't exist")
+
+
+def test_save_analysis(tmpdir):
+    ch_num = 94
+    ch = dummy_channel(ch_num=ch_num)
+    ch2 = ch.summarize_pulses().with_good_expr_pretrig_rms_and_postpeak_deriv()
+    data = mass2.Channels({ch_num: ch2}, description="dummy dataset", bad_channels={})
+    dir = pathlib.Path(tmpdir)
+    savefile = dir / "test_save"
+    actual_savefile = savefile.with_suffix(".zip")
+    print(f"{savefile=}")
```

Comment:
We should remove prints from tests before merging.

Reply:
Oops. Yes. I tend to forget print in tests, because when the test stops failing, the printed text stops showing up.